nasa/harmony
# Harmony 1419

## Jira Issue ID
HARMONY-1419

## Description
Wanted to get this PR out before Chris does :)

This adds a k8s service to schedule work-items on the service queues. This is only for local development until harmony-ci-cd is changed as well. Additional changes will be needed to support harmony-in-a-box with queues, but for now queues are disabled by default, so this is not an issue.

## Local Test Steps
1. pull this branch (harmony-1419)
1. set USE_SERVICE_QUEUES=true and NODE_ENV=production in your .env
2. run `bin/restart-services`
3. verify that the scheduler is running using `kubectl get pods -n harmony`
4. run harmony
5. issue queries

## PR Acceptance Checklist
